GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -522,16 +522,16 @@ public FileSystemTaskResult AddMissingModifiedFiles()
                     using (ITracer activity = this.context.Tracer.StartActivity(
                         nameof(this.indexParser.AddMissingModifiedFilesAndRemoveThemFromPlaceholderList),
                         EventLevel.Informational))
-                    {
```

Comment:
I'm definitely on team fix line endings!

Since there are no actual changes in this file though, I would remove it from the commit.